src/client/Client.cc

```diff
@@ -11605,6 +11605,9 @@ int Client::ll_create(Inode *parent, const char *name, mode_t mode,
 		      int flags, struct stat *attr, Inode **outp, Fh **fhp,
 		      int uid, int gid)
 {
+  if (fhp)
+    *fhp = NULL;
```

Comment:
Yeah, now that you mention it, I don't see any callers that pass a NULL pointer in here. I'll change it to initialize it unconditionally.